src/Views/Form/Templates/Sequoia/assets/css/field-api.scss

```diff
@@ -0,0 +1,121 @@
+/*
```

Comment:
Can you rename this file `legacy-consumer.scss`? These styles are for the legacy consumer specifically and not the Fields API in general. This distinction is important, because the goal of the Fields API is that people can write their own consumers.

Reply:
fixed: https://github.com/impress-org/givewp/compare/748f4a489867...d40fd1bf23cb